addons/misc/functions/fnc_suitcaseNukeZeus.sqf

```diff
@@ -0,0 +1,211 @@
+                    deleteVehicle _device;
+                };
+                case "smoke_yellow": {
+                    private _smoke = createVehicle ["SmokeShellYellow_Infinite", getPosATL _device, [], 0, "CAN_COLLIDE"];
+                    hideObjectGlobal _smoke;
+                    _device setVariable [QGVAR(suitcaseNuke_smoke), _smoke];
+                    _smoke attachTo [_device];
+                    _device addEventHandler ["Deleted", {
+                        params ["_device"];
+                        deleteVehicle (_device getVariable [QGVAR(suitcaseNuke_smoke), objNull]);
+                    }];
+                };
+                default {};
+            };
+        };
+
+        // private _customCode = _device getVariable [QGVAR(suitcaseNuke_code), ""];
+        // if(_customCode isNotEqualTo "") then {
+        //     private _codeTarget = _unit getVariable [QGVAR(suitcaseNuke_codeTarget), 0];
+
+        //     if((_codeTarget == 0 and hasInterface) || {
+        //         (_codeTarget == 1 and isServer) || {
+        //         (_codeTarget == 2)}}
+        //     ) then {
+        //         [_device] call (compile _customCode);
+        //     };
+        // };
+    }];
+
+
+    // Defusal Code
+    if(_defuser > 0) then {
+
+        // TODO: with ace, also (optionally) require DefusalKit item
+        private _ace = (isClass(configFile >> "CfgPatches" >> "ace_main"));
+        private _condition = switch(true) do {
+            case (_defuser == 1 and _ace) : { " and ([player] call ace_common_fnc_isEOD)" }; // EOD
+            case (_defuser == 1 and !_ace) : { " and (player getUnitTrait ""ExplosiveSpecialist"")" }; // EOD
+            case (_defuser == 2 and _ace) : { " and (([player] call ace_common_fnc_isEngineer) or ([player] call ace_common_fnc_isEOD)" }; // Engineer
+            case (_defuser == 2 and !_ace) : { " and ((player getUnitTrait ""Engineer"") or (player getUnitTrait ""ExplosiveSpecialist""))" }; // Engineer
+            default { "" }; // Anyone
+        };
+
+
+        [
+            _device,
+            "Defuse Device",
+            "\a3\ui_f_oldman\data\IGUI\Cfg\holdactions\repair_ca.paa",
+            "\a3\ui_f_oldman\data\IGUI\Cfg\holdactions\repair_ca.paa",
+            format ["(_this distance _target < 3) and (alive _target) and _target getVariable [""crowsza_misc_suitcaseNuke_isArmed"", false]%1", _condition],
+            "_caller distance _target < 3 and (alive _target) and _target getVariable [""crowsza_misc_suitcaseNuke_isArmed"", false]",
+            {},
+            {},
+            {
+                _target setVariable [QGVAR(suitcaseNuke_isArmed), false, true];
+                if (isClass(configFile >> "CfgPatches" >> "crowsEW_sounds")) then {
+                    [getPos _target, 200, "device_disarmed", 2] call crowsEW_sounds_fnc_playSoundPos;
+                    [getPos _target, 200, "futuristic_machine_turn_off", 2] call crowsEW_sounds_fnc_playSoundPos;
```

Comment:
You should use ``getPosASL``. ``PlaySound3D`` expects ASL position and that is also what is given to the zeus modules in EWAR.